src/config.py

```diff
@@ -224,9 +261,12 @@ def kafka_opts(self) -> str:
         Returns:
             String of Java config options
         """
-        opts = [f"-Djava.security.auth.login.config={self.zk_jaas_filepath}"]
+        opts = [
+            f"-Djava.security.auth.login.config={self.zk_jaas_filepath}",
+            "-Djavax.net.debug=ssl:handshake:verbose:session:keymanager:trustmanager",
```

Comment:
`kafka_opts` is being called on `KafkaAuth.add_user`. Does adding `"-Djavax.net.debug=ssl:handshake:verbose:session:keymanager:trustmanager"` to that call matter?

Reply:
It doesn't, but there wasn't a clear env-var from [`kafka-run-class`](https://github.com/apache/kafka/blob/15f8705246e094f7825b76a38d9f12f95d626ee5/bin/kafka-run-class.sh#L344) that seemed a better place for it.

I had originally put it in JMX opts, which was wrong I think. According to [where it's referenced](https://github.com/apache/kafka/blob/15f8705246e094f7825b76a38d9f12f95d626ee5/bin/kafka-run-class.sh#L245-L248) it seems like the right place.

The real value is adding it to `/etc/environment`, so that the `snap start charmed-kafka.daemon` uses it.